src/python/pants/backend/python/lint/isort/rules_integration_test.py

```diff
@@ -118,18 +118,20 @@ def test_multiple_targets(rule_runner: RuleRunner) -> None:
     assert fmt_result.did_change is True
 
 
-def test_respects_config_file(rule_runner: RuleRunner) -> None:
+@pytest.mark.parametrize(
+    "path,extra_args",
+    ((".isort.cfg", ["--isort-config=.isort.cfg"]), ("custom.ini", ["--isort-config=custom.ini"])),
```

Comment:
In the followup for config autodiscovery, we won't need to set `"--isort-config=.isort.cfg"` but we will still set `"--isort-config=custom.ini"`